mvlearn/preprocessing/repeat.py

```diff
@@ -0,0 +1,201 @@
+"""Filtering module."""
+# Copyright 2019 NeuroData (http://neurodata.io)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Authors: Pierre Ablin
+
+from sklearn.base import clone, TransformerMixin
+from sklearn.utils.validation import check_is_fitted
+
+from ..utils import check_Xs
+
+
+class RepeatTransform(TransformerMixin):
```

Comment:
I would call this

ViewTransformer

in sklearn we have FunctionTransformer etc.